utils/create_dummy_models.py

```diff
@@ -504,6 +504,27 @@ def convert_feature_extractor(feature_extractor, tiny_config):
     if to_convert:
         feature_extractor = feature_extractor.__class__(**kwargs)
 
+    # Sanity check: on tiny image feature extractors, a large image size results in slow CI -- up to the point where it
+    # can result in timeout issues.
+    if (
+        isinstance(feature_extractor, BaseImageProcessor)
+        and hasattr(feature_extractor, "size")
+        and isinstance(feature_extractor.size, dict)
+    ):
+        largest_image_size = max(feature_extractor.size["height"], feature_extractor.size["width"])
```

Comment:
This assumes that the keys in the `size` dict are `["height", "width"]`, but they could also be e.g. `["longest_edge", "shortest_edge"]`
```suggestion
        largest_image_size = max(feature_extractor.size.values())
```